packages/components/select-v2/src/defaults.ts

```diff
@@ -3,6 +3,7 @@ import { isValidComponentSize } from '@element-plus/utils/validators'
 import type { PropType } from 'vue'
 import type { ComponentSize } from '@element-plus/utils/types'
 import type { OptionType } from './select.types'
+import type { Options } from '@element-plus/components/popper/src/use-popper/index'
```

Comment:
We should avoid using this way since the build bundle would not look like this.

In this case

```ts
import type { Options } from '@popperjs/core'
```